.github/workflows/Extractor-CI.yml

```diff
@@ -0,0 +1,38 @@
+name: CI/CD for Extractor application
```

Comment:
please do not change all at ones.

update for extractor is completely unrelated and should not be in this PR.

Reply:
done